internal/repository/sqlite_repo/repo.go

```diff
@@ -0,0 +1,75 @@
+package sqlite_repo
+
+import (
+	"fmt"
+	"github.com/google/uuid"
+	"gorm.io/driver/sqlite"
+	"gorm.io/gorm"
+	"log/slog"
+	"viktig/internal/entities"
+	"viktig/internal/repository"
+	"viktig/internal/repository/in_memory_repo"
+)
+
+type postgresRepo struct {
+	db *gorm.DB
+}
+
+func New() repository.Repository {
+	db, err := gorm.Open(sqlite.Open("./dev/dev.db"), &gorm.Config{})
+	if err != nil {
+		slog.Warn(fmt.Sprintf("failed to connect to database use in-memory repo: %v", err))
+		return in_memory_repo.New()
```

Comment:
I suggest to exit the application with non-zero exit code if there are issues with connecting to the database. Using in-memory repo as a fallback will lead to data loss in production